src/components/producerData/components/AddItem.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { List, Avatar, Icon } from 'antd';
+import { LIST_DATA, PRODUCER_PAGE } from '../constants';
+
+const { EDIT, LARGE, LOGO, PRODUCT_ITEM, VERTICAL } = PRODUCER_PAGE;
+
+class AddItem extends React.Component {
+    render() {
+        return (
+            <div className={PRODUCT_ITEM}>
```

Comment:
unneeded `div` move className to `List` component

Reply:
OK